_posts/2022-06-28-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+Hello again Trino nation,
+
+We are pleased to announce the 2022 Trino Summit, which will be happening in a
+few short months on November 10th. The event will be *hybrid* this year, meaning
+you can join us online or in-person at the Commonwealth Club right on the
+waterfront in downtown San Francisco, California.
+
+The event will be hosted by Starburst Data, and if you'd like to register to
+attend (online *or* in-person), you can do so on
+[Starburst's registration page.](https://www.starburst.io/info/trinosummit/)
+
+<!--more-->
+
+Perhaps more exciting, if you'd like to submit a paper or give a talk at this
+year's Trino summit, we're putting out a call for speakers. We will be accepting
+submissions from now until September 15th, so there's plenty of time to polish
+your submission before sending it in. Once you're ready, you can submit on
+[the Sessionize page for the event.](https://sessionize.com/trino-summit-2022/)
+We're looking for intermediate to advanced-level talks on a variety of themes,
```

Comment:
Be more specific and add all the info in this post ..

Also ... add some info that is should be related to Trino and interesting to other users, contributors or others in the community

Reply:
The example list is quite long - 16 items, 76 words, it alone would be over a quarter of the post, and probably half of the "space" taken up because it'd need to be bulleted. I'd include it if it were shorter, but mentioning every suggested topic in this post is going to make the post very long-looking and unwieldy. I imagine most people attending _aren't_ going to be giving talks, so that's a huge info dump that's only relevant to a minority of readers, and even for the people who will be giving talks, I suspect that they don't need to see the list to consider whether or not their talk would be relevant to the Trino summit... and if they do, they can go to the speaker registration page, because they're going through there, anyway.